examples/data/aks.json

```diff
@@ -1,45 +0,0 @@
-{
-  "aks": {
-    "Cluster": {
-      "properties": {
-        "networkProfile": {
-          "dockerBridgeCidr": "172.17.0.1/16",
-          "dnsServiceIP": "10.0.0.10",
-          "serviceCidr": "10.0.0.0/16",
-          "podCidr": "10.244.0.0/16",
-          "networkPlugin": "kubenet"
-        },
-        "enableRBAC": true,
-        "nodeResourceGroup": "MC_preflight_preflight-test_uksouth",
-        "addonProfiles": {
-          "httpApplicationRouting": {
-            "config": null,
-            "enabled": false
-          }
-        },
-        "servicePrincipalProfile": {
-          "clientId": "***********************"
-        },
-        "agentPoolProfiles": [
-          {
-            "osType": "Linux",
-            "maxPods": 110,
-            "storageProfile": "ManagedDisks",
-            "osDiskSizeGB": 100,
-            "vmSize": "Standard_B2s",
-            "count": 1,
-            "name": "agentpool"
-          }
-        ],
-        "fqdn": "preflight-test-dns-f2de96e5.hcp.uksouth.azmk8s.io",
-        "dnsPrefix": "preflight-test-dns",
-        "kubernetesVersion": "1.14.8",
-        "provisioningState": "Succeeded"
-      },
-      "type": "Microsoft.ContainerService/ManagedClusters",
-      "name": "preflight-test",
-      "location": "uksouth",
-      "id": "/subscriptions/*****************/resourcegroups/preflight/providers/Microsoft.ContainerService/managedClusters/preflight-test"
-    }
-  }
-}
```

Comment:
Why is this being removed?

I've found it quite useful to run the [stubbed config](https://github.com/jetstack/preflight/blob/master/examples/stubbed.preflight.yaml) as a basic test of the CLI, and I imagine it could be used with the agent too.

Reply:
Ah I missed your comment in the commit message! I still feel this is useful to have...